README.md

```diff
@@ -13,115 +13,36 @@ The Interbit Platform is currently a test environment provided for information,
-```
-
-If you would like to run the sites involved in MVP development (app-account, app-project, template & redux-proxy-server)
-
-```
-$ npm run start:dev # from the root
-```
-
-If you have changed lib-react-interbit and would like to include your changes in your app...
-
-```
-$ cd packages/lib-react-interbit
-$ npm run build
-$ cd ../app-to-work-on
-$ npm start
-```
-
-If you want to work on the common components in lib-react-interbit there is a demo app that can be launched with `npm start`. The source for the demo app is in `packages/lib-react-interbit/demo/src`
-
-This system currently deploys to a heroku pipeline.
-
-CI tests all commits before auto deploying to the `development` stage of the pipeline.
-
-Deployment events can be viewed and controlled from slack in the #machines channel: https://devcenter.heroku.com/articles/chatops
-
-
-## To change the Template.
-
-There is a node script inside of utils/src/distributeTemplate.js that will distribute some of the changes made to the template files to all packages that match the pattern `/app-*/`
-
-It will clobber the following files in each app with the template versions:
- - anything in src that is a js or svg file
- - anything in the public dir
- - redux/chainReducer.js
- - services/blockchain.js
-
- Except for:
-  - src/App.js
-  - src/exports.js
-  - public/manifest.json
-
-To anything matching `/app-*/` and the template itself it will:
-  - force standardized proxying
-  - force standardized scripts
-  - ensure the following dependencies are met
-
-```json
-{
-  "interbit": "1.0.0",
-  "interbit-covenant-tools": "0.4.12",
-  "lib-react-interbit": "1.0.0",
-  "react": "^16.2.0",
-  "react-bootstrap": "0.31.5",
-  "react-redux": "5.0.6",
-  "react-scripts": "1.0.17",
-  "redux": "3.7.2",
-  "seamless-immutable": "^7.1.2"
-}
-```
+### Make the following changes in `packages/app-the-new-thing`:
 
+1. `.package.json` Change the name
+   ```
+   {
+     "name": "app-the-new-thing",
+     "version": "0.1.0",
+     ...
+   }
+   ```
 
-*If you need to make any changes to these standardized files think very carefully about doing so before propagating them out to all the app packages.*
+1. Tweak `src/App.js` so that you can identify your site
 
-### To Distribute Template Changes
+## Install the dependencies
+Run `npm i` from the repository root.  This will take care of all the lerna dependencies and set up for you
 
-Commit your intended changes and work with a clean git status... this will make resetting your changes much easier if you break something.
+## Build the new Interbit site
+1. From within the `packages/app-the-new-thing` folder, run `npm run start:interbit`.  This command will keep running until interrupted.  Let it continue to run.
+1. Open a second shell.  From within the `packages/app-the-new-thing` folder, run `npm run start`.  This runs the development server and will cause a browser window to open to http://localhost:3000 You should see your site load in the browser.
```

Comment:
Should be:
`npm run interbit:start`

It's an easy mistake to make.

Reply:
Can you please close this PR without merging?  I'll submit another

Thanks.